tests/test_black.py

```diff
@@ -1938,6 +1938,14 @@ def test_equivalency_ast_parse_failure_includes_error(self) -> None:
         err.match("invalid character")
         err.match(r"\(<unknown>, line 1\)")
 
+    def test_type_comment_syntax_error(self) -> None:
```

Comment:
Would prefer a test in test_format.py where we don't have to write a new test function.